frontend/src/layouts/MainLayout.vue

```diff
@@ -346,611 +346,585 @@
-      error = error.toString()
-      this.showNotif({
-        message: `RPC error in command '${command}': ${error}`,
-        color: 'negative'
-      })
-      this.log({
-        level: 'error',
-        message: `${this.componentName}: RPC error in command '${command}': ${error}`
-      })
-    },
+  dismissNotif = notify({
+    message: message,
+    color: color,
+    textColor: 'white',
+    position: 'bottom-right',
+    timeout: 0,
+    group: true,
+    actions: actions
+  })
+}
+
+const log = ({ level, message }) => {
+  const timestamp = Date.now()
+  const t = new Date(timestamp)
+  history.value.push({
+    level,
+    timestamp,
+    time: `${t.getHours()}:${t.getMinutes()}:${t.getSeconds()}`,
+    message
+  })
+  switch (level) {
+    case 'error':
+      logger.error(message)
+      break
+    case 'warn':
+      logger.warn(message)
+      break
+    case 'info':
+      logger.info(message)
+      break
+    case 'debug':
+      logger.debug(message)
+      break
+  }
+}
+
+const rpcErrorHandler = (error, command) => {
+  error = error.toString()
+  showNotif({
+    message: `RPC error in command '${command}': ${error}`,
+    color: 'negative'
+  })
+  log({
+    level: 'error',
+    message: `${componentName}: RPC error in command '${command}': ${error}`
+  })
+}
+
+const downloadLogs = () => {
+  let text = ''
+  for (const line of history.value) {
+    text += `${line.time} [${line.level}] ${line.message}\n`
+  }
+  const dl = document.createElement('a')
+  dl.setAttribute('download', 'logs.txt')
+  dl.setAttribute('href', 'data:text/plain,' + text)
+  dl.style.visibility = 'hidden'
+  document.body.append(dl)
+  dl.click()
+  dl.remove()
+}
+
+const start = async (manual) => {
+  const ports = await findKnownDevices()
+  if (ports && ports.length > 0) {
+    await connect()
+    setTimeout(async () => {
+      await startRpc()
+      await readInfo()
+      await setTime()
+    }, 500)
+  } else {
+    flags.value.portSelectRequired = true
+    if (manual) {
+      return selectPort()
+    }
+  }
+}
 
-    downloadLogs () {
-      let text = ''
-      for (const line of this.history) {
-        text += `${line.time} [${line.level}] ${line.message}\n`
-      }
-      const dl = document.createElement('a')
-      dl.setAttribute('download', 'logs.txt')
-      dl.setAttribute('href', 'data:text/plain,' + text)
-      dl.style.visibility = 'hidden'
-      document.body.append(dl)
-      dl.click()
-      dl.remove()
-    },
-
-    async start (manual) {
-      const ports = await this.findKnownDevices()
-      if (ports && ports.length > 0) {
-        await this.connect()
-        setTimeout(async () => {
-          await this.startRpc()
-          await this.readInfo()
-          await this.setTime()
-        }, 500)
-      } else {
-        this.flags.portSelectRequired = true
-        if (manual) {
-          return this.selectPort()
-        }
+onMounted(async () => {
+  if ($q.screen.width < 900) {
```

Comment:
In quasar, these breakpoints can be customized. In the future, you need to do this absolutely in the whole project. And in the script, use the approach `$q.screen.lt.lg`

SCSS: https://quasar.dev/style/breakpoints/
$q.screen: https://quasar.dev/options/screen-plugin/#configuration